### DISCUSSION TITLE
How to know if WebRtcStreamer is disconnected ？

### DISCUSSION THREAD

**Discussion by l329831342** [mpromonet/webrtc-streamer#662] [General]
How to know if WebRtcStreamer is disconnected due to network or other reasons? After disconnecting, the screen will come to a standstill. If the user does not distinguish carefully, they cannot know that the video has been disconnected. The experience needs to be improved. It is recommended to add a judgment parameter or callback function for WebRtcStreamer video stream disconnection

**Comment by mpromonet** [comment]
Hi @l329831342 

Sure, feel free to submit a PR to extend current behavior.
You may also use https://github.com/Eyevinn/webrtc-player.

Best Regards,
Michel.
